### ISSUE TITLE
Remove completely the Angular UI section

### ISSUE THREAD

**Issue by cescoffier** [quarkusio/quarkus-workshops#58]

**Comment by agoncal** [comment]
@cescoffier what will you use instead ? Qute templates ?

**Comment by cescoffier** [comment]
No, it’s about removing it from the document, not from the code.

**Comment by erpei** [comment]
I encountered issues in the UI setup section of the workshop project.  The setup commands like mvn install and package.sh are not working accordingly, e.g. no node_modules folder generated, I reused a node_modules generated outside, but the ng build in the package.sh complained the src code.  Worked on setup the UI for several days without success.  My suspicion is my versions in Angular are not matching with the project's UI.  Any tips and advise are greatly appreciated.

**Comment by cescoffier** [comment]
\CC @agoncal

**Comment by agoncal** [comment]
@cescoffier Now you are breaking my heart ! Angular is the only UI framework I barely understand, and you want to remove it ;o)

What do you suggest ? We could just bundle it as a ZIP, and tell the reader to unzip it and install it on Quarkus ? Or make it available on CDN ?

I agree that the UI is just there to help the visual aspect and can be discard from the doc

**Comment by cescoffier** [comment]
The zip approach looks good to me. We can still provide the source and add it to the zip we deliver. 
This issue is mostly about removing the section from the document (as it's not an angular workshop).

**Comment by agoncal** [comment]
And what about refactoring the document and move the section to an annexe? "Install the Zip file this way to have a UI. Go to the Annexe XYZ if you want to know how to develop this application" ?

**Comment by cescoffier** [comment]
Yes, that would be awesome! We don't lose content and we reduce the "workshop"

**Comment by agoncal** [comment]
Ok. I've assigned it to me (but I won't be able to do it before end-of-march)

**Comment by magicwerk** [comment]
I just spent a few hours trying to get the section 
https://quarkus.io/quarkus-workshops/super-heroes/#_installing_the_web_application_on_quarkus
to work but miserably failed with npm and angular compilation errors and trying to run package.sh under Windows.

In the end, I realized that there is already prebuilt content in the ui-super-heroes\src\main\resources\META-INF\resources directory and the UI is therefore working even if my build attempts fail.

As the workshop is about Quarkus and not building an UI with Angular, I support the request to remove the content from the workshop document - it would have saved me quite some time.
